### PR TITLE
Add two targets: SPEEDYBEEF7V3 and SPEEDYBEE_F745_AIO

### PR DESCRIPTION
**Pull request by SEA729**

### CODE REVIEW

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+serial 3 1024 115200 57600 0 115200
+
+# master
+set gyro_to_use = SECOND
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set blackbox_device = SDCARD
+set dshot_burst = ON
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:122]: Isn't this the default?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW270FLIP
+set gyro_1_align_pitch = 1800
+set gyro_1_align_yaw = 2700
+set gyro_2_spibus = 1
+set gyro_2_sensor_align = CW0FLIP
+set gyro_2_align_pitch = 1800
+
+profile 0
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:144]: What's this?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+rateprofile 1
+
+rateprofile 2
+
+rateprofile 3
+
+rateprofile 4
+
+rateprofile 5
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:160]: What's this?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+rateprofile 0
+
+rateprofile 1
+
+rateprofile 2
+
+rateprofile 3
+
+rateprofile 4
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:158]: What's this?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+profile 2
+
+rateprofile 0
+
+rateprofile 1
+
+rateprofile 2
+
+rateprofile 3
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:156]: What's this?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+profile 1
+
+profile 2
+
+rateprofile 0
+
+rateprofile 1
+
+rateprofile 2
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:154]: What's this?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+
+profile 0
+
+profile 1
+
+profile 2
+
+rateprofile 0
+
+rateprofile 1
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:152]: What's this?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+set gyro_2_sensor_align = CW0FLIP
+set gyro_2_align_pitch = 1800
+
+profile 0
+
+profile 1
+
+profile 2
+
+rateprofile 0
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:150]: What's this?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+set gyro_1_align_yaw = 2700
+set gyro_2_spibus = 1
+set gyro_2_sensor_align = CW0FLIP
+set gyro_2_align_pitch = 1800
+
+profile 0
+
+profile 1
+
+profile 2
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:148]: What's this?

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+set gyro_1_sensor_align = CW270FLIP
+set gyro_1_align_pitch = 1800
+set gyro_1_align_yaw = 2700
+set gyro_2_spibus = 1
+set gyro_2_sensor_align = CW0FLIP
+set gyro_2_align_pitch = 1800
+
+profile 0
+
+profile 1
```
Comment (DusKing1) [inline 配置/违约/SPBE-SPEEDYBEEF7V3.config:146]: What's this?

**Review by DusKing1 — CHANGES_REQUESTED** [review]

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config (hunk 2 above)
Comment (SEAFPV) [reply 配置/违约/SPBE-SPEEDYBEEF7V3.config:144]: OK,sorry...

File: 配置/违约/SPBE-SPEEDYBEEF7V3.config (hunk 1 above)
Comment (SEA729) [reply 配置/违约/SPBE-SPEEDYBEEF7V3.config:122]: The above 4 lines are from the 'diff all bare' command. The 3rd line(Line 121) is what we need for this product. And we saw the other 3 lines(Line 119, 120, and 122, all from the 'diff all bare' command) are all included in other targets submitted recently. Should we delete these three lines manually? thanks for your review.

Comment (haslinghuis) [reply 配置/违约/SPBE-SPEEDYBEEF7V3.config:122]: All settings here are defined on many other targets. Not sure which default you are referring to @DusKing1

File: configs/default/SPBE-SPEEDYBEE_F745_AIO.config [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+serial 0 64 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set current_meter = ADC
```
Comment (haslinghuis) [inline configs/default/SPBE-SPEEDYBEE_F745_AIO.config:95]: If this AIO includes ESC I was expecting 
```
set motor_pwm_protocol = DSHOT600
```

File: configs/default/SPBE-SPEEDYBEEF7V3.config [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin C08 0
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+
+# feature
+feature -RX_PARALLEL_PWM
```
Comment (haslinghuis) [inline configs/default/SPBE-SPEEDYBEEF7V3.config:106]: Obsoleted on next line

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/SPBE-SPEEDYBEE_F745_AIO.config (hunk 11 above)
Comment (SEA729) [reply configs/default/SPBE-SPEEDYBEE_F745_AIO.config:95]: thanks for your review.

File: configs/default/SPBE-SPEEDYBEE_F745_AIO.config [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+# pin E09: DMA2 Stream 3 Channel 6
+dma pin E11 1
+# pin E11: DMA2 Stream 2 Channel 6
+dma pin D12 0
+# pin D12: DMA1 Stream 0 Channel 2
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+
+# feature
+feature -RX_PARALLEL_PWM
```
[the remark this answers is not in the document]
Comment (SEA729) [reply configs/default/SPBE-SPEEDYBEE_F745_AIO.config]: thanks for your review.

**Comment by haslinghuis** [comment]
Please squash commits

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**